src/main/java/com/kanedias/vanilla/plugins/saf/SafUtils.java

```diff
@@ -25,7 +26,8 @@
     /**
      * Finds needed file through Document API for SAF. It's not optimized yet - you can still gain wrong URI on
      * files such as "/a/b/c.mp3" and "/b/a/c.mp3", but I consider it complete enough to be usable.
-     * @param currentDir - document file representing current dir of search
+     *
+     * @param currentDir            - document file representing current dir of search
```

Comment:
Don't introduce changes if they are not necessary

Reply:
gotcha